edbterraform/data/terraform/aws/modules/kubernetes/outputs.tf

```diff
@@ -0,0 +1,30 @@
+output "cluster_id" {
+  description = "EKS cluster ID"
+  value       = module.eks.cluster_id
+}
+
+output "cluster_endpoint" {
```

Comment:
This is named `host` for azure and google. This should be updated so all provider outputs match where possible.

https://github.com/EnterpriseDB/edb-terraform/blob/e9f4504d27f4c7ddd688422022d845d7ce5bea0b/edbterraform/data/terraform/azure/modules/kubernetes/outputs.tf#L26-L28

https://github.com/EnterpriseDB/edb-terraform/blob/e9f4504d27f4c7ddd688422022d845d7ce5bea0b/edbterraform/data/terraform/gcloud/modules/kubernetes/outputs.tf#L8-L11

Reply:
Updated to "host"